app/components/scroll-to-top.js

```diff
@@ -1,13 +1,49 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
-
+import { registerDestructor } from '@ember/destroyable';
 import { tracked } from '@glimmer/tracking';
+
 export default class ScrollToTopComponent extends Component {
   @service router;
   @tracked isLive = this.router.currentRoute.name === 'live';
+  @tracked isScrollToTopVisible = false;
+
+  constructor() {
+    super(...arguments);
+
+    const onBodyScroll = myDebounce(
+      this,
+      function () {
+        if (window.scrollY !== 0) {
+          this.isScrollToTopVisible = true;
+        } else {
+          this.isScrollToTopVisible = false;
+        }
+      },
+      1000,
+    );
+
+    document.body.onscroll = onBodyScroll;
```

Comment:
```suggestion
    document.body.onscroll = onBodyScroll;
```
is above line required??

Reply:
removed, forgot to remove